src/Microsoft.AspNetCore.Mvc.Razor.Extensions.Version1_X/ModelDirective.cs

```diff
@@ -79,7 +72,7 @@ protected override void ExecuteCore(RazorCodeDocument codeDocument, DocumentInte
                 var visitor = new Visitor();
                 var modelType = GetModelType(documentNode, visitor);
 
-                if (_designTime)
+                if (documentNode.Options.DesignTime)
```

Comment:
There aren't really that many places where we need to know up front if something is design time or not. This was probably the simplest part of the PR.